src/Prima/Pyxis/Form/Date.elm

```diff
@@ -491,15 +491,61 @@ taggerAttribute (Date config) =
 
 stringToDate : String -> DatePicker.Date
 stringToDate str =
+    let
+        guessedDate =
+            case String.length str of
+                6 ->
+                    let
+                        year =
+                            String.slice 4 6 str
+
+                        guessedYear =
+                            year
+                                |> String.toInt
+                                |> Maybe.map
+                                    (\y ->
+                                        if y <= 25 then
+                                            "20" ++ year
+
+                                        else
+                                            "19" ++ year
+                                    )
+                                |> Maybe.withDefault (String.slice 4 6 str)
+                    in
+                    case year of
+                        "19" ->
+                            str
+
+                        "20" ->
+                            str
+
+                        _ ->
+                            String.join "/" [ String.slice 0 2 str, String.slice 2 4 str, guessedYear ]
+
+                8 ->
```

Comment:
Tutta questa roba andrebbe riscritta un pochino meglio e magari inserita in microfunzioni